### PR TITLE
Implement token gateway

### LINKED ISSUE #41: [Backend | Database Setup] Implement Token Gateway Interface Through DynamoDB
Implement token gateway interface through DynamoDB. Requires #40.

### PR DESCRIPTION
**Pull request by Jacob4649**
Closes #41. Implements the ITokenStore interface through DynamoDB

### CODE REVIEW

File: backend/src/main/java/com/javahelp/backend/data/DynamoDBTokenStore.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    /**
+     * Creates an {@link Token} from the specified DynamoDB item
+     *
+     * @param item {@link Map} representing a DynamoDB item
+     * @return the created {@link Token}, or null if item does not satisfy
+     * conditions for a DynamoDB representation of a {@link Token}
+     */
+    private static Token fromDynamo(Map<String, AttributeValue> item) {
+
+        if (!item.containsKey("token")
```
Comment (Lam207) [inline backend/src/main/java/com/javahelp/backend/data/DynamoDBTokenStore.java:118]: Nice step!

**Review by Enkhzul001 — APPROVED** [review]
Looks good

**Review by Lam207 — APPROVED** [review]
Looks good to me!

**Review by yuanshu777 — APPROVED** [review]
Looks good!

**Pull request merged**